cl/opinion_page/views.py

```diff
@@ -387,8 +368,10 @@ def view_opinion(request, pk, _):
             "favorite_form": favorite_form,
             "get_string": get_string,
             "private": cluster.blocked,
-            "citing_clusters": citing_clusters,
-            "top_authorities": cluster.authorities[:5],
+            "top_children": children[:5],
```

Comment:
Does slicing the array like this take the full results and slice them or does it only query Solr for five results? The difference could be significant.

Reply:
Changes reverted in f85e738